integration-tests/environments/react-native/harness/runner.js

```diff
@@ -134,6 +146,46 @@ async function run(headless, spawnLogcat) {
   } else if (PLATFORM === "ios") {
     // Ask React Native to run the ios app
     rn.sync("run-ios", "--no-packager", "--simulator", IOS_DEVICE_NAME);
+  } else if (PLATFORM === "catalyst") {
+    // Ask React Native to run the ios app
+    const myMacDeviceId = xcode.getMyMacDeviceId();
+
+    // This will just build the app, but not launch it
+    rn.sync("run-ios", "--no-packager", "--udid", myMacDeviceId);
+
+    // This will retrieve the build information and launch the app
+    const buildSettings = child_process.execFileSync(
+      "xcodebuild",
+      [
+        "-workspace",
+        "./ios/RealmReactNativeTests.xcworkspace",
+        "-scheme",
+        "RealmReactNativeTests",
+        "-sdk",
+        "macosx",
+        "-configuration",
+        "Debug",
+        "-showBuildSettings",
+        "-json",
+      ],
+      { encoding: "utf8" },
+    );
+    const settings = JSON.parse(buildSettings);
+
+    for (const i in settings) {
```

Comment:
Could this potentially spawn multiple apps?

Reply:
If this xcodebuild command started returning multiple targets, yes.  But that shouldn't be possible.

Reply:
In any case, I have modified it to open from the last "app" WRAPPER_EXTENSION it finds.